pkg/tool/kubectl.go

```diff
@@ -23,6 +23,17 @@ func NewKubectl(exec exec.ProcessExecutor) Kubectl {
 // DeleteNamespace deletes the specified namespace. If the namespace does not terminate within 90s, pods running in the
 // namespace and, eventually, the namespace itself are force-deleted.
 func (k Kubectl) DeleteNamespace(namespace string) {
+
+	fmt.Println("deleting pvcs...")
```

Comment:
Match existing style: 

```suggestion
	fmt.Println("Deleting pvcs...")
```

Reply:
done